### PR TITLE
Governance: add Conway overview + diagram

### PR DESCRIPTION
**Pull request by CarlosLopezDeLara**
👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.

1. Please complete a Checklist
2. Fill in all sections of the template
3. Click "Create pull request"

## Checklist

 <-- Please fill the boxes with [x] before submitting a pull request --> 

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 
- [x] I have not committed any changes to `yarn.lock` (or have [removed these changes](https://developers.cardano.org/docs/portal-contribute/#i-committed-yarnlock-to-my-pr-branch-what-do-i-do)).

## Updating documentation or Bugfix

<-- Help us understand your motivation by explaining why you decided to make this change. We must be able to understand the purpose of your change from this description.  Does this fix a bug? Does it close an issue? If so please mention it.  -->

### CODE REVIEW

**Review by rdlrt — APPROVED** [review]

File: docs/get-started/cardano-cli/governance/governance.md [hunk 1]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
...
+7. At the next epoch boundary, the system **applies** the _enactState_.
+8. A governance action **Expires** if it does not accumulate enough votes during it's lifetime. Its last chance to be ratified is when the _DRepPulser_ talies the votes on the epoch following its expiration. 
+
+
+
+
+
+
+
+
```
Comment (rphair) [inline docs/get-started/cardano-cli/governance/governance.md:202]:
```suggestion
```

**Review by rphair — APPROVED** [review]
@CarlosLopezDeLara looks great in local build ✅ and reads well.  @Ryun1 what do you think?

File: docs/get-started/cardano-cli/governance/governance.md [hunk 2]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
 
 ## Conway era update proposals
 
-TODO: Intro and diagram similar to the ones above. 
+The Conway Era introduces a new governance framework to Cardano, marking a significant evolution in how decisions about the protocol are made. Building on the foundations laid in previous eras, Conway empowers Cardano with a decentralized governance model where decision-making is accessible to stakeholders. Key features include the introduction of *Delegated Representatives (DReps)*, the *Constitutional Committee*, and an decision making role for *Stake Pool Operators* (SPOs). These entities collaborate through a formalized voting system to vote protocol paramter changes that can be proposed by any ada holder. These governance bodies are in charge of treasury funds, adding or removing members from the constitutional committee, making updates to the Cardano Constitution. Conway era makes Cardano more resilient and 100% community-driven.
```
Comment (Ryun1) [inline docs/get-started/cardano-cli/governance/governance.md:108]:
```suggestion
The Conway Era introduces a new governance framework to Cardano, marking a significant evolution in how decisions about the protocol are made. Building on the foundations laid in previous eras, Conway empowers Cardano with a decentralized governance model where decision-making is accessible to stakeholders. Key features include the introduction of *Delegated Representatives (DReps)*, the *Constitutional Committee*, and an decision making role for *Stake Pool Operators* (SPOs). These entities collaborate through a formalized voting system to vote on governance actions that can be proposed by any ada holder. These governance bodies are in charge of treasury funds, adding or removing members from the constitutional committee, making updates to the Cardano Constitution and changes to protocol parameters. Conway era makes Cardano more resilient and 100% community-driven.
```

I think this makes more sense

File: docs/get-started/cardano-cli/governance/governance.md [hunk 3]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
...
+
+The process for submitting and voting update proposals undergoes significant changes in the Conway Era. First, Genesis delegations and MIR certificates are eliminated. In `DState`, the fields associated with these features are no longer included, and `DelegEnv` no longer contains the fields it had in the Shelley era, effectively rendering the genesis keys obsolete.
```
Comment (Ryun1) [inline docs/get-started/cardano-cli/governance/governance.md:110]:
```suggestion
The process for submitting and voting protocol update proposals undergoes significant changes in the Conway era. First, Genesis delegations and MIR certificates are eliminated. In `DState`, the fields associated with these features are no longer included, and `DelegEnv` no longer contains the fields it had in the Shelley era, effectively rendering the genesis keys obsolete. 
```

File: docs/get-started/cardano-cli/governance/governance.md [hunk 4]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
...
+
+The process for submitting and voting update proposals undergoes significant changes in the Conway Era. First, Genesis delegations and MIR certificates are eliminated. In `DState`, the fields associated with these features are no longer included, and `DelegEnv` no longer contains the fields it had in the Shelley era, effectively rendering the genesis keys obsolete. 
+
+### Governance actions
+
+In Conway era, any ada holder can submit a Governance action, which is the on-chain method to put a proposal to the consideration of the three governance bodies. There are 7 types of governance actions:
```
Comment (Ryun1) [inline docs/get-started/cardano-cli/governance/governance.md:114]:
```suggestion
In the Conway ledger era, any ada holder can submit a Governance action, which is the on-chain method to put a proposal to the consideration of the three governance bodies. There are 7 types of governance actions:
```

File: docs/get-started/cardano-cli/governance/governance.md [hunk 5]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
...
+
+| Governance Action | Description |
+| ----------------- | ----------- |
+| NoConfidence      | A motion to create a state of no-confidence in the current constitutional committee |
+| UpdateCommittee   | Changes to the members of the constitutional committee and/or to its signature threshold and/or terms |
+| NewConstitution   | A modification to the off-chain Constitution and the proposal policy script |
+| TriggerHF3        | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
```
Comment (Ryun1) [inline docs/get-started/cardano-cli/governance/governance.md:121]:
```suggestion
| TriggerHF         | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
```

File: docs/get-started/cardano-cli/governance/governance.md [hunk 6]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
...
+
+### Governance actions
+
+In Conway era, any ada holder can submit a Governance action, which is the on-chain method to put a proposal to the consideration of the three governance bodies. There are 7 types of governance actions:
+
+| Governance Action | Description |
+| ----------------- | ----------- |
+| NoConfidence      | A motion to create a state of no-confidence in the current constitutional committee |
+| UpdateCommittee   | Changes to the members of the constitutional committee and/or to its signature threshold and/or terms |
+| NewConstitution   | A modification to the off-chain Constitution and the proposal policy script |
```
Comment (Ryun1) [inline docs/get-started/cardano-cli/governance/governance.md:120]:
```suggestion
| NewConstitution   | A modification to the off-chain Constitution and the or proposal policy script |
```

File: docs/get-started/cardano-cli/governance/governance.md [hunk 7]
```diff
@@ -105,4 +105,98 @@ cardano-cli conway transaction submit --tx-file updateNOpt.tx.signed
...
+| TriggerHF3        | Triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade |
+| ChangePParams     | A change to one or more updatable protocol parameters, excluding changes to major protocol versions ("hard forks") |
```
Comment (Ryun1) [inline docs/get-started/cardano-cli/governance/governance.md:122]:
```suggestion
| ChangePParams     | A change to one or more updatable protocol parameters, excluding changes to major or minor protocol versions ("hard forks") |
```

File: docs/get-started/cardano-cli/governance/governance.md (hunk 6 above)
Comment (rphair) [reply docs/get-started/cardano-cli/governance/governance.md:120]: @Ryun1 can you double-check this suggestion?  Should it b:e _modification to the off-chain Constitution **or** the proposal policy script_?

Comment (Ryun1) [reply docs/get-started/cardano-cli/governance/governance.md:120]: oops, I meant "and or"
I will edit

Comment (rphair) [reply docs/get-started/cardano-cli/governance/governance.md:120]: aha, I didn't read it as the grammatically incorrect but universally accepted `and/or` ... which is logically just an `or` 🤓 (I've corrected the change above)

**Comment by katomm** [comment]
Great additions from @Ryun1 that are not yet included in https://developers.cardano.org/docs/portal-style-guide#editorial-style-guide. For consistency we should add them at some point.

**Pull request merged**